client/blocks/post-share/test/nudges.jsx

```diff
@@ -67,11 +68,11 @@ describe( 'UpgradeToPremiumNudgePure basic tests', () => {
 	} );
 
 	test( 'hide when user cannot upgrade', () => {
-		const props = {
+		const localProps = {
```

Comment:
Please note I had to rename this const in order for prettier to pass the commit (no shadow)